.github/workflows/playwright-docker.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - name: Get secrets from Vault
         if: inputs.secrets != ''
         id: get-secrets
-        uses: grafana/shared-workflows/actions/get-vault-secrets@get-vault-secrets/v1.2.1 # zizmor: ignore[unpinned-uses]
+        uses: grafana/shared-workflows/actions/get-vault-secrets@get-vault-secrets/v1.2.1
```

Comment:
We don't need `# x-release-please-start-version # x-release-please-end-version` comments?

Reply:
We do, check again - they should be there now